tools/ontology-builder/src/all_ontology_generator.py

```diff
@@ -245,15 +245,12 @@ def _extract_ontology_term_metadata(
         # Gets ancestors
         ancestors = _get_ancestors(onto_term, allowed_ontologies)
 
-        # Special Case: skip the current term if it is an NCBI Term, but not a descendant of 'NCBITaxon:33208'.
+        # Special Case: skip the current term if it is an NCBI Term, but not a descendant of 'NCBITaxon:33208' (Animal)
```

Comment:
nit: NCBITaxon:33208 is [Metazoa](https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCBITaxon_33208)

Reply:
yeah i know, i just put in "animal" because that was the "plain english" synonym listed here:

![image](https://github.com/user-attachments/assets/3e2799aa-c0f6-4d7b-821e-37fe90e0f332)

Reply:
oh sorry--i thought it was a comment to document the label rather than a note for clarity